RIOT-OS/RIOT
# tests/driver_bq2429x: move test to test-with-config

### Contribution description

This PR moves the `tests/driver_bq2429x` test script to test-with-config and removes the blacklist on CI (not needed anymore). The README of the application is also updated to explain how to run the test script.




### Testing procedure

`make -C tests/driver_bq2429x test` should do nothing, so not fail. To actually perform this test, ensure a device is correctly plugged and configured, and run (untested as I don't have that hardware:

```
make BOARD=<a board> -C tests/driver_bq2429x flash test-with-config
```




### Issues/PRs references

This was missed in #15771.
